ci-operator/config/openshift-eng/agent-qe-infra/tools/analyze_jobs.py

```
@@ -0,0 +1,316 @@
+
+    baremetal_jobs = []
+
+    for yaml_file in Path(base_dir).rglob("*.yaml"):
+        # Skip non-config files
+        if yaml_file.name in ['OWNERS', 'README'] or 'tools' in str(yaml_file):
+            continue
+
+        try:
+            with open(yaml_file, 'r') as f:
+                content = yaml.safe_load(f)
+        except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)